build.sbt

```diff
@@ -0,0 +1,58 @@
+import Dependencies._
+
+lazy val spark = "2.0.2"
+lazy val si = spark match {
+  case "2.2.0" => SparkInfo("0.10.4", "0.13.1")
+  case "2.1.0" => SparkInfo("0.10.4", "0.12")
+  case "2.0.2" => SparkInfo("0.10.3", "0.11.2")
+}
+
+lazy val root = (project in file(".")).
+  settings(
+    inThisBuild(List(
+      organization := "is.hail",
+      scalaVersion := "2.11.8",
+      version      := "0.2.0-SNAPSHOT"
+    )),
+    name := "hail",
+    spName := "hail-is/hail",
+    sparkVersion := spark,
+    sparkComponents ++= Seq("sql", "mllib"),
```

Comment:
These two `spark` variables are added by the `sbt-spark-package` sbt plugin from `projects/plugins.sbt`. Without that plugin it's kind of a huge PITA to use Spark with sbt.

Reply:
more details: https://github.com/databricks/sbt-spark-package